GUI_intro.java

```diff
@@ -0,0 +1,16 @@
+import javax.swing.JOptionPane;
+
+public class GUI_intro {
+        public static void main(String[] args) {
+        String name = JOptionPane.showInputDialog("Enter your name");
+        JOptionPane.showMessageDialog(null, "Hello "+name);
```

Comment:
```suggestion
        JOptionPane.showMessageDialog(null, "Hello " + name);
```
C'est plus sympa d'aérer un peu autours des opérateurs ^^